docs/send-data/kubernetes/collecting-kubernetes-metrics.md

```diff
@@ -0,0 +1,61 @@
+---
+id: collecting-kubernetes-metrics
+title: Collecting Kubernetes Metrics
+sidebar_label: Collecting Kubernetes Metrics
+description: Collecting Kubernetes Metrics
+---
+
+By default, we collect selected metrics from the following Kubernetes components:
+
+- `Kube API Server` configured with `kube-prometheus-stack.kubeApiServer.serviceMonitor`
+- `Kubelet` configured with `kube-prometheus-stack.kubelet.serviceMonitor`
+- `Kube Controller Manager` configured with `kube-prometheus-stack.kubeControllerManager.serviceMonitor`
+- `CoreDNS` configured with `kube-prometheus-stack.coreDns.serviceMonitor`
+- `Kube EtcD` configured with `kube-prometheus-stack.kubeEtcd.serviceMonitor`
+- `Kube Scheduler` configured with `kube-prometheus-stack.kubeScheduler.serviceMonitor`
+- `Kube State Metrics` configured with `kube-prometheus-stack.kube-state-metrics.prometheus.monitor`
+- `Prometheus Node Exporter` configured with `kube-prometheus-stack.prometheus-node-exporter.prometheus.monitor`
+
+If you want to forward additional metric from one of these services, you need to edit the corresponding Service Monitor definition. Service
+Monitor tells Prometheus which metrics it should take from the service.
+
+## Example
```

Comment:
This example doesn't work anymore, as otel collector doesn't collect cAdvisor metrics based on a ServiceMonitor. We should change this to kube-state-metrics imo.